src/comp/Makefile

```
@@ -1,5 +1,6 @@
 PWD:=$(shell pwd)
 TOP:=$(PWD)/../..
+WANT_TCL=yes
 
 include $(TOP)/platform.mk
```

```suggestion

WANT_TCL=yes
include $(TOP)/platform.mk
```

I put this above because I thought it was the same category of thing as `$(TOP)`, which is also used by `platform.mk`.